GhostNetwork.Gateway/Users/ISecuritySettingsResolver.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Domain;
+using System;
+using System.Threading.Tasks;
+
+namespace GhostNetwork.Gateway.SecuritySettings;
+
+public interface ISecuritySettingsResolver
```

Comment:
what’s the reason for this interface?

Reply:
Describing of functionality for any resolvers. Remove it and write resolvers without any interface?

Reply:
Yes, remove it. You don’t use this interface at all